src/index.html

```diff
@@ -20,8 +20,692 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <header class="header">
+        <div class="header__box">
+          <a
+            class="header__logo"
+            href="index.html"
+          >
+            <img
+              src="./images/logo.png"
+              alt="logo"
+              class="header__img"
+            />
+          </a>
+          <nav class="nav">
+            <ul class="nav__menu">
+              <li class="nav__item">
+                <a
+                  href="#"
```

Comment:
Using `href="#"` is not recommended as it can cause unexpected behavior when clicked. Consider using a valid URL or `javascript:void(0);` if the link is meant to be handled by JavaScript.